spec/integ/matrix-client-syncing.spec.js

```diff
@@ -42,12 +42,24 @@ describe("MatrixClient syncing", function() {
             presence: {},
         };
 
+        it("should start keepalives if a /sync hits a timeout error", function(done) {
+            httpBackend.when("GET", "/sync").respond(200, syncData).timeout(61*1000);
+            client.startClient();
+            spyOn(client._syncApi, '_startKeepAlives').andCallThrough();
+            // Work-around since
+            // Jasmine expect(<method>).toThrow() seems to break self-references in
+            // that method.
+            httpBackend.flush().done(function() {
+                done();
```

Comment:
Missing assertion on `_startKeepAlives`? To check if it has/has not been called?